requirements.txt

```diff
@@ -1,5 +1,7 @@
 boto3>=1.4.0
 botocore>=1.4.34
-PyYAML>=3.12
-jsonschema>=2.5.1
 ipaddress>=1.0.16
+jsonschema>=2.5.1
+PyYAML==3.11
```

Comment:
and since skew is not a dependency, we shouldn't be pinning pyyaml to an older release.